docs/configuration/experimental-configuration.rst

```diff
@@ -0,0 +1,50 @@
+.. _sec:cfg-participant-experimental:
+
+===================================================
+Experimental Configuration
+===================================================
+
+.. |ProductName| replace:: SIL Kit
+
+.. |br| raw:: html
```

Comment:
This probably breaks every other generator if we'd ever start using them. Consider using the [line block syntax](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks).

Reply:
I was trying to get a nice line break before the ".. note", no success with line block syntax. However not so importent, I'll remove the |br| and live with the small visual flaw.